crates/context/src/lib.rs

```diff
@@ -928,6 +959,20 @@ impl ContextManager {
                 context_id
             );
         };
+
+        let member_id = signer_id.rt().expect("infallible conversion");
```

Comment:
localize as much of these as possible, to the call position, (so long as it's not reused)

transmutation is mostly useful for specialization, so in most cases, it can be reduced to just the caller

Reply:
Updated